test/test-creation.js

```diff
@@ -973,8 +973,8 @@ describe('JHipster generator', () => {
                 `${TEST_DIR}features/user/user.feature`
             ]);
             assert.noFile([
-                `${TEST_DIR}gatling/gatling.conf`,
-                `${TEST_DIR}gatling/logback.xml`
+                `${TEST_DIR}gatling/conf/gatling.conf`,
```

Comment:
Yes indeed, tests were wrong. I thought I had moved them, but no I only made a refactoring of these tests during my PR.
Anyway, now it's ok